### DISCUSSION TITLE
ItemPermission::addPermission should be requirePermission?

### DISCUSSION THREAD

**Discussion by dhb52** [orchidsoftware/platform#1520] [General]
`ItemPermission::addPermission` should be `ItemPermission::requirePermission`?

**Comment by tabuna** [comment]
Can you explain it? We create a group and add permission to this group.

```php
ItemPermission::group(__('Systems'))
    ->addPermission('platform.systems.roles', __('Roles'))
    ->addPermission('platform.systems.users', __('Users')),
```

That is, at this moment, we still do not require anything.

**Comment by jonatemps** [comment]
how to display my permissions on the screen ?
